Maths/CountDigit.js

```diff
@@ -1,14 +1,12 @@
 /*
-
-    1. This program will ask a user to enter a number
-    2. Then it will count the number of digits present in the given number
-    3. Will print the number of digits in console
-
+1. This program will ask a user to enter a number
+2. Then it will count the number of digits present in the given number
+3. Will print the number of digits in console
 */
 
 function CountDigit (number) {
-    var digits = Math.floor(Math.log10(number) + 1);
-    return digits;
+  var digits = Math.floor(Math.log10(number) + 1)
+  return digits
 }
 
-console.log('The number of digits in the number: ' + CountDigit(parseInt(prompt('Enter number: '))));
```

Comment:
@Utsav1999, thank you for your contribution 

this is from the library function
`
const prompt = require('prompt-sync')();`
this line is required
we are using nodejs to run and test code.
prompt is a lib function to take input, _window.prompt_ will not work in node
**to install**
`npm install prompt-sync`

[resource](https://www.codecademy.com/articles/getting-user-input-in-node-js#:~:text=Once%20you've%20loaded%20the,Hey%20there%20%24%7Bname%7D%60) 

@itsvinayak